### PR TITLE
gubernator: add more patterns to match

### PR DESCRIPTION
**Pull request by kargakis**
Adds `panic:` for now

### CODE REVIEW

File: gubernator/regex.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def wordRE(word):
 # HACK: match ANSI colored lines by allowing preceding "m",
 # as in"\x1b[0;31mFAILED\x1b[0m"
 
-default_words = ["build timed out", "error", "fail", "failed", "fatal", "undefined"]
+default_words = ["build timed out", "error", "fail", "failed", "fatal", "undefined", "panic:", ".go:[0-9]*"]
```
Comment (dims) [inline gubernator/regex.py]: Should there be a `\` before the `.`?

Comment (kargakis) [reply gubernator/regex.py]: Yeah, updated

**Comment by dims** [comment]
LGTM, no clue about about the gubernator job failure...

**Comment by kargakis** [comment]
Me neither :/

File: gubernator/regex.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def wordRE(word):
 # HACK: match ANSI colored lines by allowing preceding "m",
 # as in"\x1b[0;31mFAILED\x1b[0m"
 
-default_words = ["build timed out", "error", "fail", "failed", "fatal", "undefined"]
+default_words = ["build timed out", "error", "fail", "failed", "fatal", "undefined", "panic:", "\.go:[0-9]*"]
```
[the remark this answers is not in the document]
Comment (stevekuznetsov) [reply gubernator/regex.py]: Can you do this in this PR please

**Comment by stevekuznetsov** [comment]
Gubernator failure is from linting (highlighting THIS might be nice too)

```
W0905 12:55:41.992] + ./lint.sh
I0905 12:55:52.636] ************* Module gubernator.regex
I0905 12:55:52.636] C: 28, 0: Line too long (109/100) (line-too-long)
I0905 12:55:52.637] W: 28, 0: Anomalous backslash in string: '\.'. String constant might be missing an r prefix. (anomalous-backslash-in-string)
```

File: gubernator/regex.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kargakis) [reply gubernator/regex.py]: I thought your PR hadn't merged. I can try.

[the remark this answers is not in the document]
Comment (stevekuznetsov) [reply gubernator/regex.py]: Might be nice even to have a mapping of job or job and log name --> highlighted phrases so we can be more targeted with highlights. Unclear. That may be over-engineering this, but as in the log for the failed Gubernator test for this PR the highlights etc just distract from the real failures and led the two of you not to diagnose it correctly.

[the remark this answers is not in the document]
Comment (kargakis) [reply gubernator/regex.py]: Certainly, all the "error" and "fail" instances are distracting.

[the remark this answers is not in the document]
Comment (stevekuznetsov) [reply gubernator/regex.py]: > Certainly, all the "error" and "fail" instances are distracting.

+10

**Comment by rmmh** [comment]
The panic: keyword is good.

The `.go:\d*` keyword will cause too many false positives, since almost all of our logging notes source lines with .go in them: [search this for .go:](https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce/13289/build-log.txt).

We should wrap failed compilations somehow to extract the error better.

**Comment by kargakis** [comment]
Darn... OK, updated to include only "panic:" and opened a follow-up issue: https://github.com/kubernetes/test-infra/issues/4402

**Pull request merged**